src/ToolBox/SOS/Strike/strike.cpp

```diff
@@ -6801,6 +6823,39 @@ class CNotification : public IXCLRDataExceptionNotification4
         return S_OK;
     }
 
+    STDMETHODIMP OnCodeGenerated2(IXCLRDataMethodInstance* method, CLRDATA_ADDRESS nativeCodeLocation)
```

Comment:
It would be nice but not required to somehow avoid duplicating the code in OnCodeGenerated/OnCodeGenerated2. The only difference seems to be the call to UpdateKnownCodeAddress.